introduction.md

```diff
@@ -1,4 +1,5 @@
-I am the new student
-update line
-Ruby
-
+Deadly Skill
+===
+* Ruby
+* Rails
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?